ros_ign_bridge/src/convert/sensor_msgs.cpp

```diff
@@ -413,7 +413,7 @@ convert_ign_to_ros(
   convert_ign_to_ros(ign_msg.header(), ros_msg.header);
   convert_ign_to_ros(ign_msg.field_tesla(), ros_msg.magnetic_field);
 
-  // magnetic_field_covariance is not supported in Ignition::Msgs::Magnetometer.
+  // magnetic_field_covariance is not supported in ignition::Msgs::Magnetometer.
```

Comment:
🤔

Reply:
I'll migrate all ignition:: to gz:: even though we're not migrating the namespaces in source